README.md

```diff
@@ -108,6 +108,50 @@ HTML styled "just right".
 Jekyll will print a local URL where the site can be viewed (usually,
 [http://0.0.0.0:4000/](http://0.0.0.0:4000/)).
 
+### Testing using Docker environment 
+
+#### Build environment
```

Comment:
We can get rid of this section header, since we don't need to go that deep and there's only one section here now. We can revisit later, if needed.

```suggestion
```